verify.go

```diff
@@ -363,3 +367,19 @@ func unmarshalAttribute(attrs []attribute, attributeType asn1.ObjectIdentifier,
 	}
 	return errors.New("pkcs7: attribute type not in attributes")
 }
+
+func calculateHash(hasher Hasher, hashFunc crypto.Hash, content []byte) (computed []byte, err error) {
+	if hasher != nil {
+		computed, err = hasher.Hash(hashFunc, bytes.NewReader(content))
+	} else {
+		if !hashFunc.Available() {
+			return nil, fmt.Errorf("hash function %q not available", hashFunc.String())
```

Comment:
Apparently `Hash.String()` does not exist in go 1.14: https://pkg.go.dev/crypto@go1.14
This is causing tests to fail.

@hslatman Should I remove the usage?

Reply:
Ah, that's unfortunate. Let's make it `%v` with `hashFunc` then.